### PR TITLE
[WIP] feat: #4427 cdk deployment

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4427

### CODE REVIEW

File: packages/elements/src/components/status-indicator/__styles__/index.ts [hunk 1]
```diff
@@ -44,4 +44,10 @@ export const ElStatusIndicator = styled.span`
     color: var(--intent-danger-text);
     outline-color: var(--intent-danger-dark);
   }
+
+  &.shape-tag {
```
Comment (willmcvay) [inline packages/elements/src/components/status-indicator/__styles__/index.ts:48]: Can we add `elShapeTag` as an exported class from elements and bump a minor version please?

**Review by willmcvay — COMMENTED** [review]
Just one small thing - also, be good if @joshbalfour could have a look at the CDK code :-)

File: packages/elements/src/components/status-indicator/__styles__/index.ts (hunk 1 above)
Comment (bashleigh) [reply packages/elements/src/components/status-indicator/__styles__/index.ts:48]: sure!

File: packages/deployment-service/src/utils/project-dir.ts [hunk 2]
```diff
@@ -4,6 +4,9 @@ import { PipelineEntity } from './../entities'
 export const projectDir = (dir: string, pipeline: PipelineEntity) =>
   resolve(dir, `${(pipeline.repository as string).split('/').pop()}-master`)
 
-export const developerDir = (pipeline: PipelineEntity): string => `/mnt/efs1/project/${pipeline.developerId}/`
+const startDir = 'mnt/efs1'
+// const startDir = 'tmp'
```
Comment (joshbalfour) [inline packages/deployment-service/src/utils/project-dir.ts:8]: needed still?

**Review by joshbalfour — APPROVED** [review]
LGTM 😄

File: packages/deployment-service/src/utils/project-dir.ts (hunk 2 above)
Comment (bashleigh) [reply packages/deployment-service/src/utils/project-dir.ts:8]: Sort of, use /tmp for local testing. Quite annoying but don't really want to add an env for this just yet

**Pull request merged**